sites/frontend/components/Footer/index.js

```diff
@@ -0,0 +1,235 @@
+            title: 'Digital newspaper archive',
+            url: 'https://theguardian.newspapers.com/',
+        },
+        {
+            title: 'Complaints and corrections',
+            url: '/info/complaints-and-corrections',
+        },
+    ],
+    [
+        {
+            title: 'All topics',
+            url: '/index/subjects/a',
+        },
+        {
+            title: 'All contributors',
+            url: '/index/contributors',
+        },
+        {
+            title: 'Modern Slavery Act',
+            url:
+                '/info/2016/jul/27/modern-slavery-and-our-supply-chains?INTCMP=NGW_FOOTER_UK_GU_MODERN_SLAVERY_ACT',
+        },
+        {
+            title: 'Facebook',
+            url: 'https://www.facebook.com/theguardian',
+        },
+        {
+            title: 'Twitter',
+            url: 'https://twitter.com/guardian',
+        },
+    ],
+];
+
+const Footer = styled('footer')({
+    backgroundColor: '#333',
+    color: '#dcdcdc',
+    fontFamily: textSans,
+    fontSize: '14px',
+
+    a: {
+        color: '#dcdcdc',
+        textDecoration: 'none',
+    },
+
+    '.footer-links': {
+        display: 'flex',
+        flexWrap: 'wrap',
+        flexDirection: 'row',
+        borderTop: '1px solid #434343',
+
+        [tablet]: {
+            borderTop: 'none',
+        },
+
+        a: {
+            paddingBottom: '12px',
+            display: 'block',
+        },
+
+        ul: {
+            width: '50%',
+            borderLeft: '1px solid #434343',
+
+            [until.tablet]: {
+                ':nth-child(odd)': {
+                    borderLeft: 0,
+                    paddingLeft: 0,
+                },
+
+                ':nth-child(3)': {
+                    paddingTop: 0,
+                },
+                ':nth-child(4)': {
+                    paddingTop: 0,
+                },
+            },
+
+            [until.leftCol]: {
+                ':nth-child(1)': {
+                    borderLeft: 0,
+                    paddingLeft: 0,
+                },
+            },
+
+            [tablet]: {
+                margin: '0 10px 36px 0',
+                flex: '1 0 0',
+            },
+
+            padding: '12px 0 0 10px',
+        },
+    },
+
+    '.email-sub__iframe': {
+        paddingTop: '12px',
+
+        [leftCol]: {
+            float: 'left',
+            width: '300px',
+            marginRight: '180px',
+        },
+    },
+
+    '.footer-inner': {
+        paddingBottom: '6px',
+    },
+
+    '.copyright': {
+        fontSize: '12px',
+        padding: '6px 0 18px',
+        borderTop: '1px solid #434343',
+        marginTop: '12px',
+    },
+});
+
+type Props = {
+    links: Array<Array<Link>>,
+};
+
+const FooterLinks = ({ links }: Props) => {
+    const linkGroups = links.map(linkGroup => {
+        const ls = linkGroup.map(l => (
+            <li>
+                <a href={l.url}>{l.title}</a>
+            </li>
+        ));
+
+        return <ul>{ls}</ul>;
+    });
```

Comment:
Do we need to specify [`key`](https://reactjs.org/docs/lists-and-keys.html#keys)s for these elements?

Reply:
Yes!